src/module.d.ts

```diff
@@ -400,7 +400,7 @@ export namespace plugins {
     var PositionPlugin : (settings: PositionSettings) => GriddlePlugin;
 }
 
-export const ColumnDefinition;
-export const RowDefinition;
+export const ColumnDefinition : any;
+export const RowDefinition : any;
```

Comment:
We can't `export const columnDefinition : ColumnDefinition` ColumnDefinition must be a type.